### PR TITLE
docs: add SGLang-v0.2 slides and update README

### PR DESCRIPTION
**Pull request by zhyncs**

### CODE REVIEW

**Review by merrymercy — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
 # sgl-learning-materials
+
 Materials for learning SGLang
+
+## Blog
+
+[2024-09-04] [SGLang v0.3 Release: 7x Faster DeepSeek MLA, 1.5x Faster torch.compile, Multi-Image/Video LLaVA-OneVision](https://lmsys.org/blog/2024-09-04-sglang-v0-3/)
+
+[2024-08-15] [Supercharge Your LLMs Using SGLang For Inference: Why Speed and Efficiency Matter More Than Ever](https://blog.runpod.io/supercharge-your-llms-using-sglang/)
```
Comment (merrymercy) [inline README.md]: Remove this as it does not meet our blog's quality standard.

Comment (zhyncs) [reply README.md]: ok

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+
+[2024-07-25] [Achieving Faster Open-Source Llama3 Serving with SGLang Runtime (vs. TensorRT-LLM, vLLM)](https://lmsys.org/blog/2024-07-25-sglang-llama3/)
+
+[2024-02-05] [Fast JSON Decoding for Local LLMs with Compressed Finite State Machine](https://lmsys.org/blog/2024-02-05-compressed-fsm/)
+
+[2024-01-17] [Fast and Expressive LLM Inference with RadixAttention and SGLang](https://lmsys.org/blog/2024-01-17-sglang/)
+
+## Slides
+
+[SGLang v0.2: Faster Interface and Runtime for LLM Inference](slides/SGLang-v0.2.pptx)
```
Comment (merrymercy) [inline README.md:17]: Did you get this from Ying?

Comment (zhyncs) [reply README.md:17]: From WeChat Group

**Pull request merged**